registry/garbage-collection.md

```diff
@@ -72,15 +72,15 @@ collection. Layer `a` had one reference removed but will not be garbage
 collected as it is still referenced by manifest `A`. The blob representing
 manifest `B` will also be eligible for garbage collection.
 
-After garbage collection has been run manifest `A` and its blobs remain.
+After garbage collection has been run, manifest `A` and its blobs remain.
 
 ```
 A -----> a
     \--> b
 ```
 
 
-## How Garbage Collection works
+## How Garbage Collection works?
```

Comment:
"How garbage collection works" is a sentence fragment. I would not have used something like "Why garbage collection" either, I probably would have called it "About garbage collection". Feel free to change it.